include/seqan3/core/concept/all.hpp

```diff
@@ -14,7 +14,7 @@
 
 #include <seqan3/core/concept/cereal.hpp>
 #include <seqan3/core/concept/core_language.hpp>
-#include <seqan3/core/concept/tuple.hpp>
+#include <seqan3/utility/tuple/concept.hpp> //can be removed without test failure.
```

Comment:
```suggestion
#include <seqan3/utility/tuple/concept.hpp>
```

true, but `all.hpp` should include all headers

Reply:
ok :)